app/views/camaleon_cms/admin/settings/_file_system_settings.html.erb

```diff
@@ -20,13 +20,13 @@
             <%= label_tag t('camaleon_cms.admin.settings.filesystem_region', default: 'filesystem_region') %> <small>(*)</small><br>
             <%= text_field :options, :filesystem_region, :class => "form-control required", value: @site.get_option("filesystem_region"), placeholder: 'us-west-2' %>
         </div>
-        <!--<div class="form-group">
-            <%#= label_tag t('camaleon_cms.admin.settings.filesystem_s3_endpoint', default: 'filesystem_s3_endpoint') %> <small>(*)</small><br>
-            <%#= text_field :options, :filesystem_s3_endpoint, :class => "form-control required", value: @site.get_option("filesystem_s3_endpoint"), placeholder: 's3-us-west-2.amazonaws.com' %>
-        </div>-->
+        <div class="form-group">
```

Comment:
Thanks for the PR, @graysonchen !

Let's ask first @owen2345 why the code has been commented from the very creation? Are there any obstacles to enable it?

Reply:
AFAIR, the implementation was not complete and thus I preserved the HTML part commented.